look-and-feel/css/src/Title/Title.scss

```
@@ -28,6 +29,16 @@
     gap: 0.25rem;
   }
 
+  &--xl .af-title__title {
```

```suggestion
  &--xl &__title {
```
Applicable sur tout le fichier